naughty/debian-testing/2220-libvirt-crashes-on-startup

```diff
@@ -0,0 +1,5 @@
+# testBasicWheelUserUnprivileged (__main__.TestMachinesLifecycle)
+*
+#0 * n/a (libvirt_driver_qemu.so + *)
```

Comment:
Seems a bit brittle to rely on this, but we can delete this line if/when it gets in the way.